spec/actors/curation_concerns/apply_order_actor_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+
+describe CurationConcerns::Actors::ApplyOrderActor do
+  describe '#update' do
+    let(:curation_concern) { create(:work_with_two_children, user: user) }
+
+    let(:user) { create(:admin) }
+
+    before do
+      allow(described_class).to receive(:update).with(attributes)
```

Comment:
`update` is an instance method, so I don't think this is doing anything. Delete?